internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestActiveActiveIngestion.java

```diff
@@ -379,6 +407,24 @@ private void runSamzaStreamJob(String storeName, Time mockedTime, int numPuts, i
     }
   }
 
+  private void produceRecordWithLogicalTimestamp(
+      String storeName,
+      int index,
+      long logicalTimestamp,
+      boolean isDeleteOperation) {
+    Map<String, String> samzaConfig = getSamzaConfig(storeName);
+    VeniceSystemFactory factory = new VeniceSystemFactory();
+    try (
+        VeniceSystemProducer veniceProducer = factory.getClosableProducer("venice", new MapConfig(samzaConfig), null)) {
```

Comment:
can we reuse the producer from existing samza jobs ?

Reply:
Either way is fine, if we want to do this way then we will get a producer and use it all the way in the test.